examples/mlflow-3/sklearn_autolog.py

```diff
@@ -0,0 +1,24 @@
+# ruff: noqa
+"""
+python examples/sklearn_autolog.py
+"""
+
+import logging
```

Comment:
@harupy I ran the following example code. Generally seemed to work, but the metrics are missing from the model.

```
# ruff: noqa
"""
python examples/sklearn_autolog.py
"""

import logging
import os
import tempfile

from sklearn.datasets import load_iris
from sklearn.linear_model import LogisticRegression
from sklearn.model_selection import train_test_split
import mlflow

os.environ["MLFLOW_AUTOLOGGING_TESTING"] = "true"

mlflow.sklearn.autolog()

X, y = load_iris(return_X_y=True)
X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)

model = LogisticRegression()
with mlflow.start_run():
    model.fit(X_train, y_train)

outputs = mlflow.get_run(mlflow.last_active_run().info.run_id).outputs
print(outputs)

model_id = outputs.model_outputs[0].model_id
print(model_id)

print(mlflow.get_logged_model(model_id))
```

```
<RunOutputs: model_outputs=[<ModelOutput: model_id='beb58294-3181-4380-b5b3-77f21e4975d6', step=0>]>
beb58294-3181-4380-b5b3-77f21e4975d6
<LoggedModel: artifact_location='file:///Users/corey.zumar/mlflow/mlruns/0/models/beb58294-3181-4380-b5b3-77f21e4975d6/artifacts', creation_timestamp=1727241286667, experiment_id='0', last_updated_timestamp=1727241288677, metrics=[], model_id='beb58294-3181-4380-b5b3-77f21e4975d6', model_type=None, name='model', params={'C': '1.0',
 'class_weight': 'None',
 'dual': 'False',
 'fit_intercept': 'True',
 'intercept_scaling': '1',
 'l1_ratio': 'None',
 'max_iter': '100',
 'multi_class': 'auto',
 'n_jobs': 'None',
 'penalty': 'l2',
 'random_state': 'None',
 'solver': 'lbfgs',
 'tol': '0.0001',
 'verbose': '0',
 'warm_start': 'False'}, run_id='df30a6d2091a4ba2be7bdf536d1552b0', status='ModelStatus.READY', status_message=None, tags={}>
```

Reply:
https://github.com/mlflow/mlflow/pull/13236/commits/2636db78bdaedac19ada1249378f56c0d1bec44e fixed this issue